File: 백준/Bronze/2566. 최댓값/최댓값.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split('\n')
// let input = fs.readFileSync("./EX.txt").toString().trim().split('\n')

let maxVal = 0
let ij = [1, 1]
for (let i = 0; i < 9; i++) {
    let line = input[i].split(" ")   
    line.forEach((x, idx) => {
            if (parseInt(x) > maxVal) {
                maxVal = parseInt(x) 
                ij[0] = i+1
                ij[1] = idx+1
            }
        }
    )
}
console.log(maxVal)
console.log(...ij)

File: 백준/Bronze/2566. 최댓값/최댓값.py
import sys
input = sys.stdin.readline

N = 9
max_value = 0
ij = [1, 1]

for i in range(1, N+1):
    j = 1
    for x in input().split():
        if max_value < int(x):
            max_value = int(x)
            ij = [i, j]
        j += 1

print(max_value)
print(*ij)
